reporter/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -624,7 +624,9 @@ <h3 class="">Packages</h3>
               </dl>
               <em>Effective License:</em>
               <dl>
-                <dd>GPL-2.0-only WITH Classpath-exception-2.0 AND BSD-3-Clause AND LicenseRef-test-Apache-2.0-multi-line AND LicenseRef-test-Apache-2.0-single-line OR MIT AND LicenseRef-test-Apache-2.0-multi-line AND LicenseRef-test-Apache-2.0-single-line</dd>
+                <dd>
```

Comment:
Commit message: "Add ..."

Reply:
Sorry, more commit message nits:

> Every license needs to be linked individually and to avoid trailing white space an iterator with a while loop was used.

This again is a mix of a (good) high-level explanation with implementation details. I'd usually let the reviewer get the implementation details (for simple changes) from the code itself, so that you don't have to adjust the commit message each time you change the implementation during the review. How about just:

```
The concluded and effective licenses are SpdxExpressions, so every license
needs to be linked individually.
```